web/src/app/projects/tokens/create-edit/create-edit.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="mat-content" fxLayout="column" fxLayoutGap="20px">
+  <form (submit)="save()" [formGroup]="tokenForm">
+    <mat-card>
+      <mat-card-header>
+        <mat-card-title> <span *ngIf="!isEdit">Create</span> <span *ngIf="isEdit"> Edit </span> Token</mat-card-title>
+      </mat-card-header>
+      <mat-divider></mat-divider>
+
+      <mat-card-content fxLayout="column" fxLayoutGap="20px">
+
+        <mat-form-field>
+          <input matInput type="text" placeholder="Title" formControlName="name" required>
+          <mat-error *ngIf="tokenForm.get(['name']).hasError('required')">This field is required.</mat-error>
+          <mat-error *ngIf="tokenForm.get(['name']).hasError('minlength')">Entered text is too short.</mat-error>
+          <mat-error *ngIf="tokenForm.get(['name']).hasError('maxlength')">Entered text is too long.</mat-error>
+          <mat-error *ngIf="tokenForm.get(['name']).hasError('tokenTaken')">Entered token is taken.</mat-error>
```

Comment:
I did not see this error appear? The `create` button is disabled but with no message to the user